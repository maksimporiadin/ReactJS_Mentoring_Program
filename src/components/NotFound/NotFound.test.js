import 'jsdom-global/register';
import React from 'react';
import { NotFound } from './index';
import { shallow } from 'enzyme';

describe('NotFound', () => {
    let wrapper;

    beforeEach(()=>{
        jest.resetAllMocks();
        wrapper = shallow(<NotFound />);
    });

    it('is renders', () => {
        expect(wrapper).toMatchSnapshot();
    });
});