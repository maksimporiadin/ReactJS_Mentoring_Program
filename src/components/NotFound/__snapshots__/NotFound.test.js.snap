// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound is renders 1`] = `
<div
  className="notFound-container"
>
  Page doesn\`t Found
  <Button
    name="Back to Main Page"
    onClick={[Function]}
  />
</div>
`;
